pyproject.toml

```diff
@@ -18,39 +18,40 @@ classifiers = [
         'Operating System :: MacOS :: MacOS X',
         'Operating System :: Unix',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.6',
-        'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
-numpy = "^1.19.5"
-pillow = "^8.3.2"
-smqtk-core = ">=0.18.0"
+python = "^3.8"
+pillow = ">=9.2.0"
+numpy = ">=1.22"
+smqtk-core = ">=0.19"
 smqtk-dataprovider = ">=0.16.0"
 
 
 [tool.poetry.dev-dependencies]
 # CI
-flake8 = "^3.9.0"
-flake8-mutable = "^1.2.0"
-mypy = "^0.812"
+flake8 = ">=3.9.0"
+flake8-mutable = ">=1.2.0"
+mypy = ">=0.812"
 # Docs
 # - Also see: `docs/readthedocs-reqs.txt` for use by RTD
-Sphinx = "^3.5.3"
-sphinx-rtd-theme = "^0.5.1"
-sphinx-argparse = "^0.2.5"
-sphinx-prompt = "^1.4.0"
-livereload = "^2.6.3"
+Sphinx = ">=3.5.3"
+sphinx-rtd-theme = ">=0.5.1"
+sphinx-argparse = ">=0.2.5"
+sphinx-prompt = ">=1.4.0"
+livereload = ">=2.6.3"
 # Testing
 coverage = "^5.5"
-mock = "^4.0.3"
-pytest = "^6.2.2"
-pytest-cov = "^2.11.1"
+mock = ">=4.0.3"
+pytest = ">=7.2.0"
+pytest-cov = ">=2.11.1"
+requests = ">=2.31.0"
```

Comment:
Looks like a new dependency. Is this required due to some ripple effect?

Reply:
The change from ^ to >= is to allow versions for python3.11, the requests import is a mistake I'll remove.